cfpb/sbl-frontend
# [Page] Warnings - Phase 2, Point of Contact - Phase 1

Closes #445

## Changes
- Consolidates Warnings phase 1 with Warnings skeleton

## Questions
- Are my design assumptions correct?
  - Is the loading state on the `Save` button okay?
  - Is only showing the success alert after the user returns to the page okay (i.e. After the backend has saved their verification)? Or should it be shown as soon as the user checks the box?
  - Is disabling the checkbox for already verified submission okay?

## How to test this PR
Note: These steps demoed in the video below
1. Upload `sbl-validations-pass.csv` from the `regtech-data-validations` repo (or any file with warnings)
2. Navigate to the `Warnings` page
3. Observe that warnings were found and the alert is shown
4. Scroll to bottom of page and check the verification box
5. Click `Save and continue`, after a brief pause you should move to the POC page
6. Return to the `Warnings` page and observe that a `Success` alert is shown in place of the `warning` alert, indicating that the warnings are verified.
7. Scroll to bottom of page and observe that the checkbox is disabled, but checked 
8. Observe that the `Save and continue` button is enabled, click it. 
9. There should be no pause (no API call when Submission already verified), just a quick navigation to the next page.

## Screenshots

https://github.com/cfpb/sbl-frontend/assets/2592907/28534b49-4cf4-4c14-888d-7d8bc0ee013a

## Note
- Includes Point of Contact from https://github.com/cfpb/sbl-frontend/pull/453/
